src/_assets/stylesheets/_variables.scss

```diff
@@ -16,14 +16,14 @@ $blue-light:                #e2f4fd;
 $brand-primary:             #1155cc;
 
 // Blue color taken from gmail's stylesheet.
-$default-link:              #1155cc;
-$default-link-visited:      #1155cc;
+$default-link:              #0175C2;
```

Comment:
The comment above says this blue color is taken from gmail's stylesheet. Is that really true?

Reply:
Good catch. Removed.

Reply:
Oops, I just noticed that I forgot to change `$brand-primary` to also be #0175C2. I've done that now. I've refreshed the staging server. PTAL (the color is very similar).

Reply:
(You'll likely have to empty the browser cache for the new CSS to load.)